### ISSUE TITLE
Time tracking panel progress bars are hard to read/inaccessible color contrasts 

### ISSUE THREAD

**Issue by hjdarnel** [runelite/runelite#11915]
**Describe the bug**
The color of the green progress bar in the farming tab, for instance, has too little contrast between the "filled" and "unfilled" parts of the progress bar.

Currently the contrasts are 1.86:1 (measured with [this tool](https://webaim.org/resources/contrastchecker/)) and the [accessibility guidelines](https://www.w3.org/TR/WCAG20/) for the web require a contrast ratio of 3:1 or 4.5:1 depending on the content

I think the background of the unfinished portion of the bar should be a darker color.

**Comment by Nightfirecat** [comment]
Duplicate of #8262
